package.json

```diff
@@ -10,7 +10,8 @@
     "build:devenv": "webpack --mode production --env devenv",
     "patch": "npx mversion patch",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "lint": "lint-staged"
```

Comment:
Looks like you've run this command, but it ignored configuration of .eslintrc.json. Unfortunately, it leaded to all these changes